cv32e40s/tests/programs/custom/clic/clic.c

```diff
@@ -0,0 +1,573 @@
+//
+// Copyright 2022 Silicon Labs, Inc.
```

Comment:
Does it not need some OpenHW name in the copyright list too?
(I have no idea what the rules are.)

Reply:
That is not the case for our rtl repositories - @MikeOpenHWGroup can you comment on this? 

As this is not derivative work, I do not really see why that is needed

Reply:
The OpenHW Group does not need to hold the copyright on any code in our open source repositories, as long as the copyright holder assigns one of the premissive licenses we accept: EPL-2, Apache 2 or Solderpad 2.

We do have a lot of files in which multiple companies hold the copyright, but that is not required.  My guess is that either myself or SteveR created a header that attributed copyright to multiple organisations and it has been copied onto multiple files.